code/modules/hydroponics/seed_datums.dm

```diff
@@ -8,9 +8,9 @@
 /datum/seed/cotton/New()
 	..()
 	set_trait(TRAIT_HARVEST_REPEAT,1)
-	set_trait(TRAIT_MATURATION,6)
-	set_trait(TRAIT_PRODUCTION,6)
-	set_trait(TRAIT_YIELD,6)
+	set_trait(TRAIT_MATURATION,1)
+	set_trait(TRAIT_PRODUCTION,2)
+	set_trait(TRAIT_YIELD,4)
 	set_trait(TRAIT_POTENCY,10)
```

Comment:
Not sure why you're changing the production and maturation traits on those plants?

Reply:
Hi, the production and maturation traits are changed to fit the new probabilities and cycle rates. 1 plant age averages out to 15 minutes, but as it is a probability, this can vary. Essentially, it's just balancing out.

Reply:
Alright, just curious,